Assets/HoloToolkit/Utilities/Editor/ConfigureMenu.cs

```diff
@@ -22,34 +23,186 @@ public static void ShowHelp()
+        /// <param name="camera">Camera to update</param>
+        /// <param name="output">Output log</param>
+        private static void SetCameraDefaults(Camera camera, bool isMain, StringBuilder output)
+        {
+            output.AppendLine(string.Format("     Checking camera {0} at {1}", camera.name, camera.transform.position.ToString()));
+
+            if (isMain)
+            {
+                output.Append("     Checking position:");
+                if (camera.transform.position != Vector3.zero)
+                {
+                    camera.transform.position = Vector3.zero;
+                    output.AppendLine("  Updated position");
+                }
+                else
+                {
+                    output.AppendLine("  OK");
+                }
+            }
+            else
+            {
+                output.AppendLine("     Position ignored, not the MainCamera");
+            }
+
+            output.Append("     Checking Near Clip Plane is .85:");
+            if (camera.nearClipPlane != 0.85f)
+            {
+                camera.nearClipPlane = 0.85f;
+                output.AppendLine("  Updated");
+            }
+            else
+            {
+                output.AppendLine("  OK");
+            }
+
+            output.Append("     Checking Clear Flags are Black:");
+            if (camera.clearFlags != CameraClearFlags.Color)
+            {
+                camera.clearFlags = CameraClearFlags.Color;
+                camera.backgroundColor = Color.black;
+                output.AppendLine("  Updated");
+            }
+            else
+            {
+                output.AppendLine("  OK");
+            }
+            output.AppendLine();
+        }
+
+        /// <summary>
+        /// Applies recommended scene settings to the current scenes
+        /// </summary>
+        [MenuItem("HoloToolkit/Configure/Apply HoloLens Scene Settings", priority = 0)]
+        public static void ApplySceneSettings()
+        {
+            var output = new StringBuilder();
+            ApplySceneSettingsInternal(output);
+            EditorUtility.DisplayDialog("Results", output.ToString(), "Ok");
+        }
+
+        private static bool ApplyProjectSettingsInternal(StringBuilder output)
         {
             // Build settings
-            EditorUserBuildSettings.SwitchActiveBuildTarget(BuildTarget.WSAPlayer);
-            EditorUserBuildSettings.wsaSDK = WSASDK.UWP;
-            EditorUserBuildSettings.wsaUWPBuildType = WSAUWPBuildType.D3D;
+            output.AppendLine("** Checking Build Settings for WSA");
+            bool isWsa = EditorUserBuildSettings.activeBuildTarget == BuildTarget.WSAPlayer;
+            bool isWsaSdk = EditorUserBuildSettings.wsaSDK == WSASDK.UWP;
+            bool isDirect3D = EditorUserBuildSettings.wsaUWPBuildType == WSAUWPBuildType.D3D;
+            bool isUnityCSharpProjs = EditorUserBuildSettings.wsaGenerateReferenceProjects;
```

Comment:
I also don't think that this option should be a part of this process since the generation of reference projects isn't a required or recommended step for HoloLens development. Developers should change this option via the Build Settings window to suit their needs.